visualize/two_d_plots.py

```diff
@@ -38,7 +38,7 @@ def do_pca(self, data_to_fit=None):
         self.pca = pipe
         return self
 
-    def inverse_pca(self, poi: Sequence[float], index: Sequence[str]) -> dp.EmbeddedSeries:
+    def inverse_pca(self, poi: Sequence[float], index: Sequence[str]) -> recourse_adapter.EmbeddedSeries:
```

Comment:
Nit: looks like over 80 chars.

Reply:
We can revisit this when we push for open source quality. Both core/utils.py and visualize/two_d_plots.py have low-priority TODOs requesting refactor and can be ignored for now.